docs/team/limerencee.adoc

```diff
@@ -4,61 +4,60 @@
 :stylesDir: ../stylesheets
 
 == 1. Introduction
-=== 1.1 Purpose
-This portfolio serves as a showcase of my roles and contributions to the team project `**THRIFT**`. This project was part of
-the requirements of a year two software engineering module that I have taken as an undergraduate in NUS.
+This portfolio serves as a showcase of my roles and contributions to the team project THRIFT. This project was part of
+the requirements of a year two software engineering module that I had taken as an undergraduate in NUS.
 
-=== 1.2 Project: `**THRIFT**`
-`**THRIFT**` is a finance tracker desktop application targeting money-conscious NUS students who wish to keep
+In this project, I was the team lead and was primarily in charge of spearing the direction of the THRIFT.
+My team members were: `Lee Bo Qiang` (User Experience Developer), `Melvin Ang` (Software Architect), `Lye Jian Wen` (Quality Assurance Manager) and `Kenson Oen` (Process Analyst). We all had important and major roles to play in order to ensure that THRIFT is successful.
+
+=== 1.1 Project: THRIFT
+THRIFT is a finance tracker desktop application targeting money-conscious NUS students who wish to keep
 their spending in check. It allows its users to set a budget for each month and track incoming and outgoing transactions. This would remind the user
-not to overspend as they will constantly see their remaining balance for that month. The following image is what `**THRIFT**` looks like:
+not to overspend as they will constantly see their remaining balance for that month. The following image is how THRIFT looks like:
 
-.Main window of `**THRIFT**` which shows the user's balance as well as transactions
-image::../images/Ui.png[,550]
+.Main window of THRIFT which displays the user's balance as well as their transactions
+image::../images/UGUI.png[,700]
 
-=== 1.3 Background
-`**THRIFT**` is morphed from the original https://github.com/nus-cs2103-AY1920S1/addressbook-level3[AddressBook-Level3] application which allows users to add contacts,
+=== 1.2 Background of THRIFT
+THRIFT is morphed from the original https://github.com/nus-cs2103-AY1920S1/addressbook-level3[AddressBook-Level3] application which allows users to add contacts,
```

Comment:
Perhaps consider .. " add contacts to create a personalised list" ? i believe that sounds more fluent.

Reply:
Fixed, thanks for the tips!